README.md

```diff
@@ -1,25 +1,50 @@
-# Getting Started with Create React App
+![](https://img.shields.io/badge/Microverse-blueviolet)
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+# React Todo App
 
-## Available Scripts
+A React App that keeps track of your tasks and lets you add, edit, mark as completed and delete them.
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+- React
+
+## Getting Started
+
+To get a local copy, run the following command.
+
+- `git clone git@github.com:jfoyarzo/react-todo-app.git`
```

Comment:
- [ **OPTIONAL** ] What do you think about adding a second command below to help the user access the cloned directory? 💪🤔
```
1. To clone the repo run:
    git clone git@github.com:jfoyarzo/react-todo-app.git
2. To access the cloned directory run:
    cd your_repo_name
```